README.md

```diff
@@ -164,16 +161,16 @@ $ mvn diktat:check
 rule sets by editing the YAML file).
 
 ## Run with Gradle using diktat-gradle-plugin
-Requires a gradle version no lower than 5.3.
+Requires a gradle version no lower than 7.0
 
-This plugin is available since version 0.1.5. You can see how the plugin is configured in our examples: [build.gradle.kts](examples/gradle-kotlin-dsl/build.gradle.kts).
+You can see how the plugin is configured in our examples: [build.gradle.kts](examples/gradle-kotlin-dsl/build.gradle.kts).
 
 <details>
 <summary>Add this plugin to your `build.gradle.kts`:</summary>
 
 ```kotlin
 plugins {
```

Comment:
When you will be changing examples, please update: 
`apply()` with explicit id usage to 

import com.saveourtool.diktat.plugin.gradle.DiktatGradlePlugin

```
    apply<DiktatGradlePlugin>()

```

Reply:
added, but it's not required. plugin is applied by default

Reply:
@nulls have you tested your words on the example? For example: `gradle-kotlin-dsl-multiproject`. Is this really not needed?

Reply:
If I simply remove `apply`, I get the following error: 
```
* What went wrong:
Extension of type 'DiktatExtension' does not exist. Currently registered extension types: [ExtraPropertiesExtension]
```

Reply:
tested on `gradle-kotlin-dsl` -- `diktatCheck` does work. (but I have to add `settings.gradle.kts`).

But it doesn't work on multiproject in `allprojects` section. Because it does work if put it outside section `allprojects`.

Reply:
> But it doesn't work on multiproject in allprojects section. Because it does work if put it outside section allprojects. 

Then it's **required** as I told above. And as I have asked before - please fix it. 
Examples directory is a showcase for us.

Reply:
added already

Reply:
added already to `README.md`
Do we need to update `examples/gradle-kotlin-dsl/build.gradle.kts` and add extra `apply<DiktatGradlePlugin>` there?

Reply:
discussed offline: it was about examples. Need to update examples only. `README.md` contains default minimal configuration.

Need to add a small note to `README.md` about applying of plugin to multi-module projects

Reply:
updated, also updated examples by removing `id("com.saveourtool.diktat")` in subproject

Reply:
awesome, thx